docs/source/reference-core.rst

```diff
@@ -922,6 +922,7 @@ The nursery API
 
    See :meth:`~Nursery.start`.
 
+.. autoclass:: TaskStatus
```

Comment:
https://trio--2704.org.readthedocs.build/en/2704/reference-core.html#trio.TaskStatus doesn't seem very useful, add `:members:`?

Reply:
It's ... still not very useful. The class needs a docstring and the `started` needs one too.